XRTK-Core/Packages/com.xrtk.core/Editor/ControllerPopupWindow.cs

```diff
@@ -130,6 +130,7 @@ public static void Show(MixedRealityControllerMappingProfile profile, Serialized
             }
 
             window = (ControllerPopupWindow)CreateInstance(typeof(ControllerPopupWindow));
+            window.currentControllerName = profile.ControllerType.Type.Name;
```

Comment:
We can probably pull this out. But we could potentially get a NRE on `profile.ControllerType.Type`
```suggestion
            window.currentControllerName = profile.ControllerType?.Type?.Name;
```